MoltenVK/MoltenVK/Utility/MVKEnvironment.h

```diff
@@ -96,8 +96,13 @@
  */
 #if MVK_MACOS
 #	define MVK_SUPPORT_IOSURFACE_BOOL    1
+#	define MVK_SUPPORT_INPUT_PRIMITIVE_TOPOLOGY_BOOL	1
 #endif
 
 #if MVK_IOS
 #	define MVK_SUPPORT_IOSURFACE_BOOL	(__IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_11_0)
+#	if !defined(__IPHONE_12_0)
+#		define __IPHONE_12_0 120000
+# 	endif
+#	define MVK_SUPPORT_INPUT_PRIMITIVE_TOPOLOGY_BOOL	(__IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_12_0)
```

Comment:
I actually think this check should use `MAX_ALLOWED` instead. The `MTLPrimitiveTopologyClass` enumeration should be available in that case. Also, the whole point of the `inputPrimitiveTopologyMVK` property is that we can use it regardless of whether the host supports it, and it will automatically set the real property if it's available.

Reply:
Agreed, fixed.  I'm pretty agnostic as to how this gets fixed, I just wanted Bill to be aware that it wasn't building on Xcode 9.2 and this PR does fix that in case he wants to maintain building on older versions of Xcode/older SDKs.